### DISCUSSION TITLE
Toolbar buttons

### DISCUSSION THREAD

**Discussion by cpmshaw** [IgniteUI/igniteui-blazor#14] [Q&A]
Love the new features you have included in the 21.2 release! They are much better now. Thanks.

Is there a way to add buttons to the IgbDataGridToolbar so we can add state buttons to it or other functionality? In the examples, the buttons appear above the toolbar but it would be greta to include these so the UI looks cleaner.

**Comment by brianlagunas** [comment]
Are you wanting to add generic/custom buttons in the toolbar? Or do you just want to enable more built-in grid features as buttons in the toolbar?

**Comment by cpmshaw** [comment]
Hi Brian,

I am wanting to add buttons to the toolbar like within the Angular grids please? Can you do this with the latest version or is this still not possible?

Thanks in advance,

Charlie



From: Brian Lagunas ***@***.***>
Reply to: IgniteUI/igniteui-blazor ***@***.***>
Date: Thursday, 12 May 2022 at 17:12
To: IgniteUI/igniteui-blazor ***@***.***>
Cc: Charles Shaw ***@***.***>, Author ***@***.***>
Subject: Re: [IgniteUI/igniteui-blazor] Toolbar buttons (Discussion #14)


Are you wanting to add generic/custom buttons in the toolbar? Or do you just want to enable more built-in grid features as buttons in the toolbar?

**Reply by brianlagunas** [reply]
No, you cannot add custom buttons to the grid toolbar component at this time.
